spec/lib/ansible/runner_spec.rb

```diff
@@ -8,6 +8,23 @@
   let(:manageiq_venv_path)  { "/var/lib/manageiq/venv/python3.8/site-packages" }
   let(:ansible_python_path) { "/usr/local/lib64/python3.8/site-packages:/usr/local/lib/python3.8/site-packages:/usr/lib64/python3.8/site-packages:/usr/lib/python3.8/site-packages" }
 
+  describe ".available?" do
+    before { begin; described_class.remove_instance_variable(:@available); rescue NameError; end }
+    after  { begin; described_class.remove_instance_variable(:@available); rescue NameError; end }
```

Comment:
Might be overkill but don't we have a `spec/support/` helper to clear ivars around tests? (I'm having trouble finding it but hopefully it sparks a memory for you)

Reply:
In this case, I'm ok with that ivar being cached across tests, however this specific test is testing that exact method with various values and testing the caching itself, so that's why this one is wrapped.  I can move it to clear_caches if you think that's a better choice.